adsft/extraction.py

```diff
@@ -455,9 +418,9 @@ def parse_xml(self):
 
         parsed_content = soupparser.fromstring(self.raw_xml)
 
-        # strip out the latex stuff (for now)
-        for e in parsed_content.xpath('//inline-formula'):
-            self._remove_keeping_tail(e)
+        # remove tables, formulas and figures
+        for e in parsed_content.xpath('//table | //graphic | //disp-formula | ////inline-formula | //formula | //tex-math'):

```

Comment:
This list of things that we want to remove, it could be useful if it is a configurable thing that we can change without changing the code. Thoughts?

Reply:
I'm thinking we can address this in a future PR when I have a better idea of where I want to put this and how it should be structured.